packages/hw-ledger/src/defaults.ts

```diff
@@ -17,7 +18,8 @@ export const ledgerApps: Record<string, (transport: Transport) => SubstrateApp>
   genshiro: newGenshiroApp,
   karura: newKaruraApp,
   kusama: newKusamaApp,
-  'nodle-chain': newNodleApp,
+  node: newPolkadexApp,
```

Comment:
`node` is actually the Substrate default, so should not be used. 

In this case however, the mapping internally here is only done on the key (i.e. key needs to be the same for app, Ledger version & genesis), however to determine which chain to use, the mapping is done on genesis.

So it _should_ be 100% fine to change this to `polkadex` and it it doesn't quite do what is expected, the user of the lib should map on genesis, not name.

Reply:
Thanks Jaco! By the way, if something is not working I'll talk to Polkadex team to change the node name. But it makes totally sense!

Reply:
I'll push the changes